docs/interface/dashboard.md

```diff
@@ -1,30 +1,16 @@
 # Course dashboard
 
-![Teach dashboard](/images/courses/teach-dashboard.png)
-*Teach dashboard*
+![Teach dashboard](/images/interface/content-dashboard.png)
+*Content dashboard*
 
-The landing page consists of two parts: a sortable, filterable and paginated table of course repositories and a modal to create new courses. The table has the following columns:
+The content overview page consists of two parts: a filterable and paginated table of created content and a modal to create new or import existing courses. The table has the following columns:
 
 - **Title**: the title of the course (or repository name if it is not properly built yet)
-- **Owner**: the owner of the repository. If the repository has no owner yet, you can claim it if it's yours.
+- **Status**: current state of the content (development, live)
```

Comment:
Might be useful to enumerate all the possible states, and what they mean. for example, the difference between Live and Published isn't totally intuitive.

Reply:
Do we still have a `published` state? I think it's just `development`, `soft_launch`, and `live` now for premium courses (in that order). @gor181 please confirm

Reply:
We fetch the state from main app and then we display it. The definition and state transitions are available at https://github.com/datacamp/main-app/blob/master/app/models/concerns/course_state_transitions.rb.

Afaik, published state is used for non premium courses in order to make them available on the permanent url.

Reply:
@mayrsascha ^ please confirm